_entries/3051-issues.md

```diff
@@ -6,9 +6,11 @@ parent-id: git
 number: 3051
 ---
 
-We use git issues to discous about bugs and features that we are planning to work on. An issue has two parts.
+We use github issues to discuss bugs and features
```

Comment:
Oh, never mind - we mentioned just before this that we're going to use "github" to refer to github or gitlab. :D